packages/contracts/src/dollar/facets/UbiquityPoolFacet.sol

```diff
@@ -69,6 +83,15 @@ contract UbiquityPoolFacet is IUbiquityPool, Modifiers {
         return LibUbiquityPool.getDollarPriceUsd();
     }
 
+    /// @inheritdoc IUbiquityPool
+    function getGovernancePriceUsd()
```

Comment:
please lets do USD, all upper case

Reply:
According to our code styleguide it's supposed to be `Usd`

Reply:
i think it's fine too tho, altho usually tickers symbols are always uppercase

Reply:
Let's use the style guide then.